protoc-gen-kotlin/protoc-gen-kotlin-common/src/main/kotlin/pbandk/gen/CodeGenerator.kt

```diff
@@ -1,12 +1,19 @@
 package pbandk.gen
 
-open class CodeGenerator(val file: File, val kotlinTypeMappings: Map<String, String>) {
+open class CodeGenerator(val file: File, val kotlinTypeMappings: Map<String, String>, val params: Map<String, String>) {
     protected val bld = StringBuilder()
     protected var indent = ""
 
+    protected val jsonUseProtoNames = params["json_use_proto_names"] == "true"
```

Comment:
😎

Reply:
Actually this didn't handle nulls as well as I'd hope, so needed to add an elvis...

    protected val jsonUseProtoNames = params["json_use_proto_names"]?.toBoolean() ?: false

Let me know if you prefer the previous style, otherwise i'll leave alone.

Reply:
That works. Some other ideas could be:

* `params.getOrDefault("json_use_proto_names", "false").toBoolean()`
* `params["json_use_proto_names"].orEmpty().toBoolean()`